packages/matter-node.js/src/matter/interaction/InteractionClient.ts

```diff
@@ -7,14 +7,16 @@
 import { MessageExchange } from "../common/MessageExchange";
 import { MatterController } from "../MatterController";
 import { capitalize } from "../../util/String";
-import { Attribute, AttributeJsType, Attributes, Cluster, Command, Commands, TlvNoResponse, RequestType, ResponseType, TlvSchema, TlvStream } from "@project-chip/matter.js";
+import {
+    Attribute, AttributeJsType, Attributes, Cluster, Command, Commands, TlvNoResponse, RequestType, ResponseType,
+    TlvSchema, TlvStream, InteractionProtocolStatusCode
+} from "@project-chip/matter.js";
 import { DataReport, IncomingInteractionClientMessenger, InteractionClientMessenger } from "./InteractionMessenger";
 import { ResultCode } from "../cluster/server/CommandServer";
 import { ClusterClient } from "../cluster/client/ClusterClient";
 import { ExchangeProvider } from "../common/ExchangeManager";
 import { INTERACTION_PROTOCOL_ID } from "./InteractionServer";
 import { ProtocolHandler } from "../common/ProtocolHandler";
-import { StatusCode } from "./InteractionMessages";
```

Comment:
Let's consider using `import {InteractionProtocolStatusCode as StatusCode} from "@project-chip/matter.js"` to minimize diff issues in the future.  As long as the Admin and Interaction status codes are not used in the same files this should be bueno.

Reply:
I agree, good idea, especially because I want to rename back after the code merge :-)